ERCS/erc-7817.md

```diff
@@ -0,0 +1,112 @@
+---
+eip: 7817
+title: dApp Security Policy Standard
+description: Defines a document standard for defining the expected on-chain behavior of a dApp frontend.
+author: Bernard Wagner (bernard-wagner)
+status: Draft
+discussions-to: https://ethereum-magicians.org/t/dapp-security-policy/21431
+type: Meta
+created: 2024-10-22
+---
+
+## Abstract
+
+Introduce a standardized JSON document structure that wallets can interpret to determine whether transaction signing requests should be permitted or rejected. Policy documents should be publicly discoverable using a notary service so that a policy's authenticity can be assumed even when the origin of the transaction signing request may be compromised.
+
+As an initial implementation, DNS-in-ENS can be used as the discovery mechanism as described in ENSIP-TBC.
```

Comment:
ENSIPs haven't been approved as an external resource. See [EIP-5757](https://eips.ethereum.org/EIPS/eip-5757) for details should you wish to pursue it.